spec/rspec/mocks/and_call_original_spec.rb

```diff
@@ -248,9 +248,19 @@ def instance.foo(bar: nil); bar; end
         klass.class_exec do
           private
 
-          def method_missing(name, *args)
+          if RSpec::Support::RubyFeatures.kw_args_supported?
+            binding.eval(<<-RUBY, __FILE__, __LINE__)
+            def call_method_with_kwarg(arg, kwarg:)
+              [arg, kwarg]
+            end
+            RUBY
+          end
+
+          def method_missing(name, *args, **kwargs)
             if name.to_s == "greet_jack"
               "Hello, jack"
+            elsif name.to_s == "method_with_kwarg"
+              call_method_with_kwarg(*args, **kwargs)
```

Comment:
This will also need to be wrapped to pass our build, until we release RSpec 4 we support pre keyword rubies (sorry)

Reply:
@JonRowe ah, got it, I've changed the code, and now all the new examples are evaluated conditionally 👍

Reply:
Ruby 2.0 failed because the kwarg doesn't have a default value, I changed the condition to run only when required-kwarg are available. Hope it will pass now